src/commands/server/deploy.ts

```diff
@@ -306,9 +306,6 @@ export default class Deploy extends Command {
         if (flags['starting-csv']) {
           this.error('"starting-csv" and "version" flags are mutually exclusive. Please specify only one of them.')
         }
-        if (flags['olm-channel']) {
```

Comment:
Why?

Reply:
By a mistake, will revert. Thanks.

Reply:
Well, there is a mistake in the message. Instead of `starting-csv` there should  be `olm-channel`. But since we can construct starting CSV for a channel using version, I see no point in denying it.

Reply:
The functionality is implemented [here](https://github.com/che-incubator/chectl/pull/1680/files#diff-cd7d31f9ae278a35a9874f61cb1f17509d27a3d98e4b5c59566e578b72e3d537R96-R102).